src/Components/Navbar.js

```diff
@@ -8,13 +8,27 @@ import { Menubar } from 'primereact/menubar'
 function Navbar({ items, start, end }) {
   if (!end) {
     end = (
-      <a href="https://github.com/EddieHubCommunity/LinkFree" aria-label="View source on GitHub">
-        <i className="pi pi-github" style={{ fontSize: '2em' }}></i>
-      </a>
+      <div className="p-d-flex p-jc-center p-ai-center p-pr-2">
+        <div className="p-p-1"> Contribute on </div>
+        <a
+          href="https://github.com/EddieHubCommunity/LinkFree"
+          className="p-p-1"
+        >
+          <i className="pi pi-github" style={{ fontSize: '1em' }}></i>
+        </a>
+        <div>v0.0.0</div>
```

Comment:
```suggestion
        <div>v{version}</div>
```
The actual version should be there

Reply:
> ![image](https://user-images.githubusercontent.com/30869493/141447564-6aed558b-4b73-43ba-8be6-837b28ca2fd0.png)
> 
> Duplicate information consider removing the footer

Ok, I will do that